### PR TITLE
Documenting frame routing.

### PR DESCRIPTION
**Pull request by tralves**
Adds information on how to use `$navigateTo` in a multi-frame app.

### CODE REVIEW

File: content/docs/en/routing/manual-routing.md [hunk 1]
```diff
@@ -112,6 +112,18 @@ this.$navigateTo(Detail, {
 
 For more information about the options that you can pass, see [`NavigationEntry`](https://docs.nativescript.org/api-reference/interfaces/_ui_frame_.navigationentry).
 
+#### Navigating widthin a frame
```
Comment (ikoevska) [inline content/docs/en/routing/manual-routing.md]: a typo > must be "within"

```suggestion
#### Navigating within a frame
```

File: content/docs/en/routing/manual-routing.md [hunk 2]
```diff
@@ -112,6 +112,18 @@ this.$navigateTo(Detail, {
...
+
+Each [`<Frame>`](/en/docs/elements/components/frame) element has its own navigation stack. If you are using [`multiple frames`](/en/docs/elements/components/frame#multiple-frames), you may want to specify in which frame the navigation will occur. For instance, having a button in the side bar that changes the page in the main area. You do this by adding the `frame` option:
```
Comment (ikoevska) [inline content/docs/en/routing/manual-routing.md]: No need for code formatting in the multiple frames link.

```suggestion
Each [`<Frame>`](/en/docs/elements/components/frame) element has its own navigation stack. If you are using [multiple frames](/en/docs/elements/components/frame#multiple-frames), you may want to specify in which frame the navigation will occur. For example, having a button in the side bar that changes the page in the main area. You can do this by adding the `frame` option:
```

File: content/docs/en/routing/manual-routing.md [hunk 3]
```diff
@@ -112,6 +112,18 @@ this.$navigateTo(Detail, {
...
+
+```JavaScript
+this.$navigateTo(SomeComp, {
+  frame: '<id, or ref, or instance>'
+});
+```
+
+
The value for the `frame` option can be the `<Frame>`'s `id` (e.g. `<Frame id="main-frame">`), or the `<Frame>`'s `ref` (e.g. `<Frame ref="mainFrame">`), or the `<Frame>` element instance.
```
Comment (ikoevska) [inline content/docs/en/routing/manual-routing.md]:
```suggestion
The value for the `frame` option can be one of the following:
* the `id` of the `<Frame>` component (for example: `<Frame id="main-frame">`)
* the `ref` for the `<Frame>` (for example: `<Frame ref="mainFrame">`)
* the `<Frame>` instance itself
```

**Comment by tralves** [comment]
Omg, @ikoevska, much better! Thanks!

**Comment by tralves** [comment]
I don't know what is happening in the build process, but the result is not ok. I checked the markdown several times...

**Comment by tralves** [comment]
In the preview build, as well as in my local build, the entry was missing. Also, the contributers section was messed up.

**Pull request merged**